project/dexter-core/src/resource/dexter.em

```diff
@@ -175,8 +176,11 @@ macro initDexter()
 
 	g_dexterConfig.sourceEncoding = GetReg(g_dexterConfig.sourceEncodingRegKey)
 
-	isInstalled = GetReg("g_dexterDaemon");
-	if(isInstalled == nil || isInstalled == "isInstalled" || isInstalled != 1){
+	dexterPath = GetReg(g_dexterConfig.dexterInstallationPathRegKey);

```

Comment:
A high number of Source Insight executing at the same time can execute multiple dexter-daemon in a developer's PC.